From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Mon, 15 Apr 2024 00:53:07 +0100
Subject: [PATCH] sdm845.config: enable some more qcom drivers

It would be nice to support more qualcomm platforms, let's build more of
the clock and interconnect drivers we were disabling from the defconfig.

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
(cherry picked from commit 1f3ce62bdb954da039dea6f1eee486b9b17a0541)
Link: https://gitlab.com/sdm845-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/configs/sdm845.config | 24 +-----------------------
 1 file changed, 1 insertion(+), 23 deletions(-)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index cab9154..476aab5 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -971,12 +971,6 @@ CONFIG_INTERCONNECT_IMX=n
 CONFIG_INTERCONNECT_IMX8MQ=n
 CONFIG_INTERCONNECT_QCOM_MSM8996=n
 CONFIG_INTERCONNECT_QCOM_QCS404=n
-CONFIG_INTERCONNECT_QCOM_SC7180=n
-CONFIG_INTERCONNECT_QCOM_SM8150=n
-CONFIG_INTERCONNECT_QCOM_SM8350=n
-CONFIG_INTERCONNECT_QCOM_SM8450=n
-CONFIG_INTERCONNECT_QCOM_SM8550=n
-CONFIG_INTERCONNECT_QCOM_SC8280XP=n
 CONFIG_ARCH_BCMBCA=n
 CONFIG_ARCH_NPCM=n
 CONFIG_SERIAL_BCM63XX=n
@@ -998,22 +992,6 @@ CONFIG_SND_SOC_RT5682S=n
 CONFIG_SND_SOC_WCD938X=n
 CONFIG_SND_SOC_WCD938X_SDW=n
 CONFIG_MMC_SDHCI_OF_DWCMSHC=n
-CONFIG_CROS_TYPEC_SWITCH=n
-CONFIG_SC_GCC_8180X=n
 CONFIG_IOMMU_IO_PGTABLE_DART=n
-CONFIG_INTERCONNECT_QCOM_SC8180X=n
 CONFIG_MEMORY_HOTPLUG=n
-CONFIG_MELLANOX_PLATFORM=n
-CONFIG_CHROME_PLATFORMS=n
-CONFIG_PINCTRL_SM8150=n
-CONFIG_SM_GCC_8150=n
-CONFIG_SM_GPUCC_8150=n
-CONFIG_PINCTRL_SM8350=n
-CONFIG_SM_GCC_8350=n
-CONFIG_SM_DISPCC_8450=n
-CONFIG_SM_GCC_8450=n
-CONFIG_PINCTRL_SM8550=n
-CONFIG_PINCTRL_SM8550_LPASS_LPI=n
-CONFIG_SM_DISPCC_8550=n
-CONFIG_SM_GCC_8550=n
-CONFIG_SM_TCSRCC_8550=n
+CONFIG_MELLANOX_PLATFORM=n
\ No newline at end of file
